src/routes/webhooks/apple.ts

```diff
@@ -0,0 +1,99 @@
+import { readFileSync } from 'node:fs';
+import { env } from 'node:process';
+import type { FastifyInstance, FastifyRequest } from 'fastify';
+import { logger } from '../../logger';
+import {
+  Environment,
+  SignedDataVerifier,
+} from '@apple/app-store-server-library';
+import { isTest } from '../../common';
+import { isInSubnet } from 'is-in-subnet';
+
+const loadAppleRootCAs = (): Buffer[] => {
+  if (isTest) {
+    return [readFileSync('__tests__/fixture/testCA.der')];
+  }
+
+  return [
+    readFileSync(
+      '/usr/local/share/ca-certificates/AppleIncRootCertificate.cer',
```

Comment:
this would make the whole app fail if you dont have the file? Can we make a path configurable from env?